### PR TITLE
Invert horizontal scrolling

### LINKED ISSUE #2045: Horizontal scroll is inverted
Currently the Horizontal scroll is inverted.
If you go to any code example on glitch.com or any page that is big enough to need horizontal scrolling, you can realise that the vertical scroll works as expected (in the direction you move the joystick) but the horizontal scroll is inverted (pressing right moves the scroll to left).
Tested also on Oculus Browser and it works as expected.

### PR DESCRIPTION
**Pull request by MortimerGoro**
Fixes #2045

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java [hunk 1]
```diff
@@ -843,7 +843,7 @@ void handleScrollEvent(final int aHandle, final int aDevice, final float aX, fin
             }
             if (widget != null) {
                 float scrollDirection = mSettings.getScrollDirection() == 0 ? 1.0f : -1.0f;
-                MotionEventGenerator.dispatchScroll(widget, aDevice, aX * scrollDirection, aY * scrollDirection);
+                MotionEventGenerator.dispatchScroll(widget, aDevice, -aX * scrollDirection, aY * scrollDirection);
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java]: I think we only want to invert it for the Quest joystick, I think it is correct for touch pad based controllers.

Comment (bluemarvin) [reply app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java]: I've verified that it should only be inverted for joysticks. Not sure where that fix belongs. Our horizontal scrolling on Go is the same as the Oculus browser.

Comment (MortimerGoro) [reply app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java]: Done. I also verified that WebVR axes matches the Oculus behavior

File: app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp [hunk 2]
```diff
@@ -1025,7 +1025,7 @@ struct DeviceDelegateOculusVR::State {
         trackpadY = controllerState.inputState.Joystick.y;
         axes[0] = trackpadX;
         axes[1] = -trackpadY; // We did y axis intentionally inverted in FF desktop as well.
-        controller->SetScrolledDelta(controllerState.index, trackpadX, trackpadY);
+        controller->SetScrolledDelta(controllerState.index, -trackpadX, trackpadY);
```
Comment (bluemarvin) [inline app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp]: It's weird that we invert `Y` for immersive and `X` for browsing.

**Pull request merged**